src/resdk/resources/relation.py

```diff
@@ -77,9 +88,20 @@ def collection(self, payload):
         """Set collection to which relation belongs."""
         self._resource_setter(payload, Collection, "_collection")
 
+    @property
+    def descriptor_schema(self):
+        """Get descriptor schema."""
+        return self._descriptor_schema
+
+    @descriptor_schema.setter
+    def descriptor_schema(self, payload):
+        """Set descriptor schema."""
+        self._resource_setter(payload, DescriptorSchema, "_descriptor_schema")
+
     def update(self):
         """Clear cache and update resource fields from the server."""
         self._samples = None
+        self._descriptor_schema = None
```

Comment:
Why only clear `_descriptor_schema` here and no `_descriptor`?

Reply:
Field `descriptor_schema` is a property, and the contents of it are stored in `_descriptor_schema` attribute. Field `descriptor` is a totally "normal" attribute, thus handling / caching of these values are handled differently. See for example, how we handle descriptor schema in `Data` or `Collection` Resource - it is the same as introduced in this commit.